Entrance/Banned/details.json

```diff
@@ -0,0 +1,5 @@
+{
+	"name": "Banned",
+	"description": "Moved the 'News' panel outside the screen and centered the 'Main' panel.",
+	"creator_id": "Radorei#2773"
```

Comment:
Your `creator_id` should be your discord user ID. You can find it by enabling discord developer mode and clicking on your user icon > copy ID